features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1804,3 +1804,33 @@ CREATE TABLE IF NOT EXISTS AM_USER (
     PRIMARY KEY(USER_ID)
 );
 -- End of API-MGT Tables --
+
```

Comment:
-- End of API-MGT Tables -- comment should come to the end. also, add a new line to the end of the file

Reply:
done